Tests/RxAlamofireTests/RxAlamofireTests.swift

```diff
@@ -6,14 +6,18 @@ import RxCocoa
 import RxSwift
 import XCTest
 
-@testable import Alamofire
```

Comment:
As mentioned, this is probably able to be resolved if you install Carthage dependencies with debug config.

Reply:
But is it really necessary? We are not testing Alamofire target right?

Reply:
Ah I see - we are not utilising any internal functions in tests right now. It is alright to ignore it for now.

Reply:
Ok. Thanks